src/Sentry/Internal/NetFxInstallationsEventProcessor.cs

```diff
@@ -0,0 +1,26 @@
+#if NETFX
+using System;
+using System.Collections.Generic;
+using Sentry.Extensibility;
+using Sentry.PlatformAbstractions;
+
+namespace Sentry.Internal
+{
+    internal class NetFxInstallationsEventProcessor : ISentryEventProcessor
+    {
+        internal static readonly string NetFxInstallationsKey = ".NET Framework";
+
+        private readonly Lazy<IEnumerable<FrameworkInstallation>> _netFxInstallations =
+            new Lazy<IEnumerable<FrameworkInstallation>>(() => FrameworkInfo.GetInstallations());
```

Comment:
You need to consider multiple threads will capture events concurrently.

Use [ExecutionAndPublication](https://docs.microsoft.com/en-us/dotnet/api/system.threading.lazythreadsafetymode?view=netcore-3.1#System_Threading_LazyThreadSafetyMode_ExecutionAndPublication) here.

Another point is error handling. You need to make sure an exception thrown in `Lazy` ctor doesn't crash the whole process (or in this case since we'd handle it in the SDK, drop the whole event).

```
As stated earlier, a Lazy<T> object always returns the same object or value that it was initialized with, and therefore the Value property is read-only. If you enable exception caching, this immutability also extends to exception behavior. If a lazy-initialized object has exception caching enabled and throws an exception from its initialization method when the Value property is first accessed, that same exception is thrown on every subsequent attempt to access the Value property
```

We need to enable such caching, and when trying to access `Value` do that under a try catch.
If an exception is thrown, log that with `options.DiagnosticLogger?.LogError` and flip a flag in the object (a field) so that you don't try to access `Value` again. From that point a `LogDebug` is enough just to note that we skipped the integration.